.circleci/config.yml

```diff
@@ -1168,6 +1168,71 @@ jobs:
                       make postman
             - notify-on-failure
 
+    publish_rpm_packages:
+        parameters:
+            dry_run:
+                default: true
+                type: boolean
+            graviteeio_version:
+                type: string
+                default: $GRAVITEEIO_VERSION
+        
+        machine:
+            image: ubuntu-2004:202010-01
+            resource_class: medium
+        environment:
+            GRAVITEEIO_VERSION: << parameters.graviteeio_version >>
+        steps:
+            - checkout
```

Comment:
I think we don't need any code from the monorepo so no need to checkout

Reply:
removed